app/timeline-menu/timeline-menu.ts

```diff
@@ -0,0 +1,63 @@
+let _ = require('lodash');
+
+export enum TimelineMenuRange { LastWeek, LastMonth, LastYear, LastDecade, PreDecade, CustomRange }
+
+export class TimelineMenu {
+  LIMIT_NUM = 8640000000000000;
```

Comment:
Use this instead:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

Reply:
unfortunately this is impossible because i need the **max date number** and not the **max integer number**..
`
new Date(Number.MAX_SAFE_INTEGER) // Invalid Date
`